package.json

```diff
@@ -7,10 +7,15 @@
   },
   "scripts": {
     "start": "electron .",
-    "build": "webpack"
+    "build": "webpack",
+    "dev": "webpack --watch"
```

Comment:
It would be good to document this new command in the scripts section of the README.

Reply:
The new command is documented in README.